src/src_/security.toit

```diff
@@ -46,6 +57,7 @@ class Security0_ implements Security:
   version -> int:
     return SecSchemeVersion-SecScheme0
 
+/** This security mode 2 use SRP6a + AES-GCM. */
```

Comment:
Drop `This` unless there are other security 2 modes.

Reply:
done.